fluid-music/src/fluid/plugin-tequaliser.js

```diff
@@ -0,0 +1,747 @@
+   * @param {number} [pluginId] - optional index of the plugin to select
+   */
+  select(pluginId) {
+    return fluid.plugin.select('#TEqualiser', 'vst', pluginId);
+  },
+
+  /**
+   * Zero the selected `#TStereo Delay` plugin, resetting to sensible defaults.
+   */
+  zero() {
+    return [
+      pluginTEqualiser.setBand1State(0),
+      pluginTEqualiser.setBand2State(0),
+      pluginTEqualiser.setBand3State(0),
+      pluginTEqualiser.setBand4State(0),
+      pluginTEqualiser.setBand5State(0),
+      pluginTEqualiser.setBand6State(0),
+      pluginTEqualiser.setBand7State(0),
+      pluginTEqualiser.setBand8State(0),
+      pluginTEqualiser.setOutputGainDbfs(0),
+      pluginTEqualiser.setSoloGainDbfs(0),
+      pluginTEqualiser.setSoloBand(-1),
+    ]
+  },
+
+  /**
+   * @param {number} db Wet signal level in DBFS (0 = unity gain)
+   * @param {number} [timeInWholeNotes] time to insert automation point in
+   *    quarter notes. If no time is supplied, set the initial value
+   * @param {number} [curve=0] A number from [-1, 1] (inclusive), which
+   *    represents the curvature of the line formed by this point and the next
+   *    point. Zero implies a linear change. Higher values create a curve that
+   *    begins slowly and accelerates. Lower values create a curve that begins
+   *    quickly, and decelerates.
+   */
+  setDryLevelDbfs(db, timeInWholeNotes, curve) {
+    return fluid.plugin.setExternalParamHelper('Dry Level', db2Level(db), timeInWholeNotes, curve);
+  },
+
+  /**
+   * @param {number} db Wet signal level in DBFS (0 = unity gain)
+   * @param {number} [timeInWholeNotes] time to insert automation point in
+   *    quarter notes. If no time is supplied, set the initial value
+   * @param {number} [curve=0] A number from [-1, 1] (inclusive), which
+   *    represents the curvature of the line formed by this point and the next
+   *    point. Zero implies a linear change. Higher values create a curve that
+   *    begins slowly and accelerates. Lower values create a curve that begins
+   *    quickly, and decelerates.
+   */
+  setWetLevelDbfs(db, timeInWholeNotes, curve) {
+      return fluid.plugin.setExternalParamHelper('Wet Level', db2Level(db), timeInWholeNotes, curve);
+  },
+
+  /**
+   * @param {number} value an integer between 0-2 to set the parameter to
+   * @param {number} [timeInWholeNotes] time to insert automation point in
+   *    quarter notes. If no time is supplied, set the initial value
+   * @param {number} [curve=0] A number from [-1, 1] (inclusive), which
+   *    represents the curvature of the line formed by this point and the next
+   *    point. Zero implies a linear change. Higher values create a curve that
+   *    begins slowly and accelerates. Lower values create a curve that begins
+   *    quickly, and decelerates.
+   */
+  setBand1State(value, timeInWholeNotes, curve) {
+    return fluid.plugin.setExternalParamHelper('Band 1 State', value, timeInWholeNotes, curve);
```

Comment:
`value` is not actually updated before it is pased to `setExternalParamHelper`\

- state 0 - "remove" (parameter = 0)
- state 1 - "active" (parameter = 0.5)
- state 2 - "inactive" (parameter = 1)

Reply:
Would be nice to have an additional function `setBand1Active(true/false)` for all bands